

arch/ia64/kernel/smpboot.c:511:17: #error Fixme: Dunno how to build CPU-to-node 
map.

Seems that CONFIG_NUMA requires CONFIG_ACPI_NUMA.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/ia64/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN arch/ia64/Kconfig~ia64-config_apci_numa-fix arch/ia64/Kconfig
--- 25/arch/ia64/Kconfig~ia64-config_apci_numa-fix	Tue Jan 11 15:46:54 2005
+++ 25-akpm/arch/ia64/Kconfig	Tue Jan 11 15:47:03 2005
@@ -167,6 +167,7 @@ config NUMA
 	bool "NUMA support"
 	depends on !IA64_HP_SIM
 	default y if IA64_SGI_SN2
+	select ACPI_NUMA
 	help
 	  Say Y to compile the kernel to support NUMA (Non-Uniform Memory
 	  Access).  This option is for configuring high-end multiprocessor
_
